interfaces/python/CMakeLists.txt

```diff
@@ -131,6 +135,10 @@ else()
 
             set(CMAKE_SWIG_FLAGS "-py3")
 
+            if("${SWIG_VERSION}" VERSION_GREATER "4.0.0")
```

Comment:
pretty sure this will work like
```
if (SWIG_VERSION VERSION_GREATER 4.0.0)

Reply:
Resolved in e050c2c3a756d8e4a22f034a919b2b4a3e7cba69.